### ISSUE TITLE
CRLF lines show as modified lines

### ISSUE THREAD

**Issue by naim94a** [zed-industries/community#2365]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

After cloning a git repository, I realized that files with CRLF line endings always show as modified lines (yellow marker on the left), regardless if they were modified or not (`git diff` output is empty).

### Environment

Zed: v0.118.1 (Zed)
OS: macOS 14.2.1
Memory: 36 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
I would've guessed that Zed changed these these to `LF`, but you have no changes at all on your end?

**Comment by naim94a** [comment]
Yup, No modification to the file and it shows that all lines were modified. 
All lines are converted to LFs if I actually modify the file.
